sdcm/nemesis.py

```diff
@@ -3946,8 +3946,12 @@ def _add_new_node_in_new_dc(self) -> BaseNode:
             scylla_yml.rpc_address = new_node.ip_address
             scylla_yml.seed_provider = [SeedProvider(class_name='org.apache.cassandra.locator.SimpleSeedProvider',
                                                      parameters=[{"seeds": self.tester.db_cluster.seed_nodes_ips}])]
+            if scylla_yml.endpoint_snitch.endswith("GossipingPropertyFileSnitch"):
+                rackdc_value = 'dc=add_remove_nemesis_dc'
```

Comment:
this will work only if we don't already have a multi-DC configuration, right?

Reply:
there's another problem - we cannot set same value twice, need to fix it.

Reply:
Maybe switch to the remote_file implementation, which I think uses a dict so we can overwrite the DC key...

Reply:
right, I adapted the code. Executed base test to verify if not breaking anything: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/lukasz/job/longevity-mini-test-1h-test/13

Looks it works.

Reply:
```
11:32:28  < t:2023-06-27 08:32:28,438 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:INFO  > Traceback (most recent call last):
11:32:28  < t:2023-06-27 08:32:28,438 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:INFO  >   File "/home/ubuntu/scylla-cluster-tests/sdcm/nemesis.py", line 4433, in wrapper
11:32:28  < t:2023-06-27 08:32:28,438 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:INFO  >     result = method(*args[1:], **kwargs)
11:32:28  < t:2023-06-27 08:32:28,438 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:INFO  >   File "/home/ubuntu/scylla-cluster-tests/sdcm/nemesis.py", line 4026, in disrupt_add_remove_dc
11:32:28  < t:2023-06-27 08:32:28,438 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:INFO  >     with wait_for_log_lines(node=new_node, start_line_patterns=["rebuild.*started with keyspaces="],
11:32:28  < t:2023-06-27 08:32:28,438 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:INFO  >   File "/usr/local/lib/python3.10/contextlib.py", line 142, in __exit__
11:32:28  < t:2023-06-27 08:32:28,438 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:INFO  >     next(self.gen)
11:32:28  < t:2023-06-27 08:32:28,438 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:INFO  >   File "/home/ubuntu/scylla-cluster-tests/sdcm/wait.py", line 147, in wait_for_log_lines
11:32:28  < t:2023-06-27 08:32:28,438 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:INFO  >     raise TimeoutError(
11:32:28  < t:2023-06-27 08:32:28,438 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:INFO  > TimeoutError: timeout occurred while waiting for start log line (['rebuild.*started with keyspaces='] on node: longevity-mini-test-fix-add--db-node-5789d2ba-5
```

Reply:
Looks the Scylla version used (2022.2.3) have different log lines for rebuild process - this wait was introduced recently and possibly that's why it is failing this way.

Besides updating cassandra-rackdc.properties works.

Reply:
@fgelcer with master:latest it is passing: https://argus.scylladb.com/test/c38d2050-1e12-479a-8942-40cfaae44859/runs?additionalRuns[]=a45a0393-888e-43d8-88df-7895b553604c